src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1

```diff
@@ -372,7 +372,7 @@ function getAssetName {
     param([string] $prefix)
 
     if ($prefix -eq $null -or $prefix -eq '') {
-        $prefix = "ps";
+        $prefix = "ps$(([guid]::NewGuid().ToString() -replace '-','')[0..9] -join '')";
```

Comment:
Why are we adding to the prefix?  The idea is that, when running live the 'getRandomItemName' invocation will add random alphanumerics

Reply:
Knowing what I know now, this doesn't make sense. At the time, I wasn't sure what changes to `getRandomItemName` would affect. So, I just changed it for getAssetName since I knew that would 'fix' my issue, which was name clashing. I'm honestly not even sure if that is an issue at the point. But, I do feel that a partial-GUID would be more reliable in terms of uniqueness.

Currently, `getRandomItemName` does not add alphanumerics, it only adds a few random numbers. Should I just move this partial-GUID to `getRandomItemName`?

Reply:
Yes, we should not have two sources of randomness.